nightly.Jenkinsfile

```diff
@@ -29,7 +29,7 @@ stage('Configure') {
 		// Minimum supported versions
 		new BuildEnvironment( dbName: 'hsqldb_2_6' ),
 		new BuildEnvironment( dbName: 'mysql_8_0' ),
-		new BuildEnvironment( dbName: 'mariadb_10_5' ),
+		new BuildEnvironment( dbName: 'mariadb_11_7' ),
```

Comment:
Please revert this. The aim of this is to test the oldest still supported version.